modules/core/src/test/java/org/apache/ignite/internal/managers/discovery/IgniteTopologyPrintFormatSelfTest.java

```diff
@@ -238,18 +242,40 @@ private void doForceServerAndClientTest(MockLogger log) throws Exception {
                     || (s.contains(">>> Number of server nodes: 2") && s.contains(">>> Number of client nodes: 3"));
             }
         }));
+
+        checkNodesAdditionalLogging(log);
+    }
+
+    /**
+     * Check nodes details in log.
+     *
+     * @param log log.
+     */
+    private void checkNodesAdditionalLogging(MockLogger log){
+        if (log.isDebugEnabled())
+            assertFalse(F.exist(log.logs(), new IgnitePredicate<String>() {
```

Comment:
can be simplified to `assertFalse(F.exist(log.logs(), s -> s.contains("nodes=[")));`